src/main/java/com/streisky/discordchatgptbot/command/ping/PingCommand.java

```diff
@@ -0,0 +1,14 @@
+package com.streisky.discordchatgptbot.command.ping;
+
+import com.streisky.discordchatgptbot.command.CommandInterface;
+import com.streisky.discordchatgptbot.message.MessageModel;
+
+import java.util.List;
+
+public class PingCommand implements CommandInterface {
+
+    @Override
+    public List<MessageModel> execute(String content) {
+        return List.of(new MessageModel("Pong!"));
```

Comment:
`Pong!` could be in a constant

Reply:
As this is a unique case, I will keep it that way.